README.md

```diff
@@ -38,3 +38,5 @@ This box has all you need to get started with our [Pet Shop tutorial](http://tru
 * __How do I use this with the EthereumJS TestRPC?__
 
     It's as easy as modifying the config file! [Check out our documentation on adding network configurations](http://truffleframework.com/docs/advanced/configuration#networks). Depending on the port you're using, you'll also need to update line 16 of `src/js/app.js`.
+
+  Testing syntax highlighting.
```

Comment:
```suggestion
```

Reply:
Good catch! I removed it and pushed the fix.